api-docs/openapi-diff.js

```diff
@@ -0,0 +1,38 @@
+/* global process */
+/* eslint import/no-nodejs-modules:0 */
+/* eslint import/no-unresolved:0 */
+
+const fs = require('fs');
+const yaml = require('js-yaml');
+
+const jsonDiff = require('json-diff');
+
+const https = require('https');
+
+let sentry_api_schema_docs;
+https.get(
+  `https://raw.githubusercontent.com/getsentry/sentry-api-schema/main/openapi-derefed.json`,
+  res => {
+    res.setEncoding('utf8');
+    let rawData = '';
+    res.on('data', chunk => {
+      rawData += chunk;
+    });
+    res.on('end', () => {
+      try {
+        const parsedData = JSON.parse(rawData);
+        sentry_api_schema_docs = parsedData;
+        // console.log('PARSED DATA: ', parsedData);
+      } catch (e) {
+        // eslint-disable-next-line no-console
+        console.error(e.message);
+      }
+    });
+  }
+);
+
+const file = 'tests/apidocs/openapi-derefed.json';
+target = yaml.safeLoad(fs.readFileSync(file, 'utf8'));
+
+// eslint-disable-next-line no-console
+console.log('JSON DIFF: ', jsonDiff.diff(sentry_api_schema_docs, target));
```

Comment:
maybe`jsonDiff.diffString` would create a better output? seems like I'm getting some weird stuff when i run it. attached, vs, what i'd expect it to look lik
<img width="676" alt="Screen Shot 2022-03-03 at 1 13 51 PM" src="https://user-images.githubusercontent.com/1976777/156654381-9461185e-8379-487a-9172-3aed51a8cedf.png">
<img width="983" alt="Screen Shot 2022-03-03 at 1 18 37 PM" src="https://user-images.githubusercontent.com/1976777/156654458-5ae841fd-fb1f-4546-84f0-8c536b6830fb.png">
e

Reply:
aha - fixed in the latest commit.

Thanks! I wasn't quite sure what i should be expecting from the output